sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -161,6 +161,7 @@ export class Application extends RecordDataModel {
     type: "enum",
     enum: AssessmentStatus,
     enumName: "AssessmentStatus",
+    nullable: false,
```

Comment:
mm, for draft application, will there be value for this filed?

Reply:
This file was wrongly changed, I will revert it. This column will be removed after the new DB structure is in place.